Packs/AtlassianConfluenceCloud/Integrations/AtlassianConfluenceCloud/AtlassianConfluenceCloud.py

```
@@ -1366,6 +1366,41 @@ def get_events(client: Client, args: dict) -> tuple[list[dict], CommandResults]:
     )
 
 
+def confluence_cloud_content_get_command(client: Client, args: dict[str, str]) -> CommandResults:
+    """
+    Execute the confluence-cloud-content-get command. Fetches content details from Confluence Cloud by content ID.
+
+    :type client: ``Client``
+    :param client: The client used to send HTTP requests to the Confluence Cloud API.
+
+    :type args: ``Dict[str, str]``
+    :param args: The command arguments provided by the user.
+
+    :return: ``CommandResults`` containing the content details.
+    :rtype: ``CommandResults``
+    """
+
+    content_id = args.get("content_id")
+    params = {"expand": "body.storage"}
+    request_url = urljoin(URL_SUFFIX.get("CONTENT"), content_id)
+    try:
+        response = client.http_request(method="GET", url_suffix=request_url, params=params)
+        response_json = response.json()
+        context = remove_empty_elements(response_json)
+        readable_hr = prepare_hr_for_content_create(response_json, "Content")
+    except Exception as e:
+        context = response_json = {"Error": str(e)}
+        readable_hr = f"**Error**: {str(e)}"
```

Instead of just changing the context, we need to raise an exception. This will return to user the error in a red color which will show that there was an error. There is no need for a Commandresult to be returned if there was an error.

---

I implemented an "errorless" approach as stated by merit-maita (see https://github.com/demisto/content/pull/40623/files#r2212978259 for further information)

---

This case is different since there was a real error, this is not the same as a case where the search found nothing (an expected behavior). Therefor, change this to raise an error.